src/components/label-pill/LabelPill.tsx

```diff
@@ -33,7 +33,7 @@ export interface LabelPillProps {
 }
 
 const LabelPillContainer = React.forwardRef((props: LabelPillProps, ref: React.Ref<HTMLSpanElement>) => {
-    const { children, type = LabelPillStatus.DEFAULT, size = LabelPillSize.REGULAR, className, ...rest } = props;
+    const { children, type = LabelPillStatus.DEFAULT, size = LabelPillSize.LARGE, className, ...rest } = props;
```

Comment:
This is changing the default pill size everywhere. Is that intentional or is there one place where we should be passing the size prop as "large"?

Reply:
The change is needed in every place where the pill is, since the size is too small. :) This will fix all

Reply:
In that case it seems like we should adjust the values of the size enums to preserve relative sizing

Reply:
We'll likely have to review the usage in a number of places to make sure there aren't layout changes that need work to fix

Reply:
We need to discuss with the design team and make any changes to accommodate the new sizes where Pill elements are used